db/migrate/20221013045158_add_sidebar_topic_destination_to_user_option.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSidebarTopicDestinationToUserOption < ActiveRecord::Migration[7.0]
+  def change
+    add_column :user_options, :sidebar_topic_destination, :integer, default: false
```

Comment:
I think we have a chance for a better name here. I like `sidebar_list_destination` better since it takes you to a topic list.

Reply:
`default: false` for an integer? this is confusing ... default should probably be nil like other user options. not sure this matters, but it is confusing.